controllers/podcast.js

```diff
@@ -70,7 +67,7 @@ const deletePodcast = async (req, res, next) => {
 }
 const updatePodcast = async (req, res) => {
   const id = parseInt(req.params.id)
-  if (getPodcastById(id)) {
+  if (await getPodcastById(id)) {
```

Comment:
My goal here was to differentiate between errors, if it reaches the catch block it will send 400 error regardless of whether its podcasts not found error or bad parameters error.
However I combined the errors and changed the tests.